frontend/src/bundles/resume/store/slice.ts

```diff
@@ -0,0 +1,35 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import {
+    type ResumeGetItemResponseDto,
+    type ResumeViewsCountResponseDto,
+} from '../types/types.js';
+import { getAllResumesByUserId, getViewsCountByUserId } from './actions.js';
+
+type State = {
+    resumeId: string;
+    resumes: ResumeGetItemResponseDto[];
+    resumeViews: ResumeViewsCountResponseDto[];
+};
+
+const initialState: State = {
+    resumeId: '',
+    resumes: [],
```

Comment:
```suggestion
    resumeId: '',
    resumes: [],
```

are you using them somewhere? as I see that on mount only `resumeViews` are fulfilled